app/app.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import logOut from './actions/logOutAction';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.onClickHandler = this.onClickHandler.bind(this);
```

Comment:
Let's checkin on a call about this line. I have something to say.. that's easier "said" than "typed"